did_parser/tests/did_url/positive.rs

```diff
@@ -312,4 +312,14 @@ test_cases_positive! {
         None,
         HashMap::new(),
         HashMap::new()
+
+    test_case22:
```

Comment:
while did is technically a did url, I would find this test more fitting under `tests/did` or perhaps append some additional path to test more didurl code paths

Reply:
Agreed, adding more tests is warranted 👍

Reply:
However, the test you are commenting on is testing a strict DID URL (see the fragment at the end).

Reply:
Added a few more tests.